nemo/collections/asr/parts/utils/rnnt_utils.py

```diff
@@ -220,3 +220,274 @@ def select_k_expansions(
             k_expansions.append([(k_best_exp_idx, k_best_exp)])
 
     return k_expansions
+
+
+class BatchedHyps:
+    """Class to store batched hypotheses (labels, time_indices, scores) for efficient RNNT decoding"""
```

Comment:
Very neat, this is done so that jit compile is happy?

Reply:
Yep) There is also a test that torch.jit is fine with this structure :)